### DISCUSSION TITLE
Typo in JobJson interface?

### DISCUSSION THREAD

**Discussion by nullndr** [taskforcesh/bullmq#1201] [General]
The `returnValue` key of a job is saved as `returnvalue` in the redis hash. 

Is it a typo?

All other keys are in camel case

Here an example

```
127.0.0.1:6379> hgetall "bull:execute bulk operation:cfdf5521-e330-4c48-b353-cee825942cec"
 1) "timestamp"
 2) "1650357820049"
 3) "parent"
 4) "{\"id\":\"40e8a266-efb4-4a52-9fdc-e8d83d64fda6\",\"queueKey\":\"bull:bulk update\"}"
 5) "returnvalue"
 6) "{\"currentBulkOperation\":{\"status\":\"COMPLETED\",\"url\":null}}"
 7) "parentKey"
 8) "bull:bulk update:40e8a266-efb4-4a52-9fdc-e8d83d64fda6"
 9) "processedOn"
10) "1650357892307"
11) "delay"
12) "0"
13) "name"
14) "execute update bulk operation"
15) "stalledCounter"
16) "1"
17) "data"
18) "{}"
19) "finishedOn"
20) "1650357893417"
21) "priority"
22) "0"
23) "attemptsMade"
24) "2"
25) "opts"
26) "{\"parent\":{\"id\":\"40e8a266-efb4-4a52-9fdc-e8d83d64fda6\",\"queue\":\"bull:bulk update\"},\"delay\":0,\"attempts\":0}"
```

I took a look at the code, and the typo should be in the [JobJson](https://github.com/taskforcesh/bullmq/blob/master/src/interfaces/job-json.ts) interface [here](https://github.com/taskforcesh/bullmq/blob/7a3206fe97cf801db4042cfe96a912c695037916/src/interfaces/job-json.ts#L15) and [here](https://github.com/taskforcesh/bullmq/blob/7a3206fe97cf801db4042cfe96a912c695037916/src/interfaces/job-json.ts#L31)

**Comment by manast** [comment]
It is not a typo. It has been since the first version and we cannot change it since it would create a breaking change that will certainly break thousands of prod deployments...

**Reply by nullndr** [reply]
The breaking change problem is exactly what I thought, will it be fixed in the next major release?

**Reply by manast** [reply]
No. I do not think it is worth it.

**Reply by nullndr** [reply]
Why not? It will help consistency

**Reply by manast** [reply]
Because it creates a big problem to all current users for something that is no issue for anybody.
